docs/routes/api/farms.md

```diff
@@ -0,0 +1,18 @@
+| Field | READ | WRITE |
+| :---- | :--- | :---- |
+| id | Yes | No |
+| ownerId | Yes | No |
+| name | Yes | Yes |
+| address | Yes | Yes |
+| website | Yes | Yes |
+| phone | Yes | Yes |
+| email | Yes | Yes |
+| subscribed | User | No |
```

Comment:
Well, this is confusing since just reading it here and not checking the actual code one would think that anyone can change the name/website/email etc of the farm, which is not the case.
One would also think that nobody can change the "subscribed" field.